lib/views/components/custom_bottom_nav_bar.dart

```diff
@@ -70,33 +64,18 @@ class CustomBottomNavBar extends ConsumerWidget {
     int index,
     int currentIndex,
   ) {
-    if (currentIndex == index) return;
-
+  //if (currentIndex == index) return;
     ref.read(currentIndexProvider.notifier).state = index;
 
-    switch (index) {
-      case 0:
-        _navigateTo(context, const CourseOverview());
-        break;
-      case 1:
-        _navigateTo(context, const DownloadedCourses());
-        break;
-      case 2:
-        _navigateTo(context, const PinnedCourses());
-        break;
-      case 3:
-        _navigateTo(context, const MyNotifications());
-        break;
-    }
   }
-
+/*
   void _navigateTo(BuildContext context, Widget page) {
     Navigator.of(context, rootNavigator: true).pushAndRemoveUntil(
       MaterialPageRoute(builder: (context) => page),
       (Route<dynamic> route) => false,
     );
   }
-
+*/
```

Comment:
Please omit to have commented-out code in the repository 😊

Reply:
done